src/commands/init.js

```diff
@@ -298,7 +309,7 @@ function askForDirectory(resolve) {
         name: 'name',
         message: 'What do you want to name the directory?'
     }], ({ name }) => {
-        const directoryPath = path.join(process.cwd(), name);
+        const directoryPath = name.indexOf('/') === 0 ? name : path.join(process.cwd(), name);
```

Comment:
That's is why I had the directory passed along to `askForDirectory` first. I removed and figured why did I ever want this here! ;) Thx. I'll fix that and also use `getAbsolutePath`!